lib/kafo/scenario_manager.rb

```diff
@@ -29,17 +29,18 @@ def available_scenarios
     end
 
     def list_available_scenarios
-      say "Available scenarios"
+      say ::HighLine.color("Available scenarios", :cyan)
       available_scenarios.each do |config_file, content|
         scenario = File.basename(config_file, '.yaml')
-        say "  #{content[:name]} (use: --scenario #{scenario})"
-        say "        " + content[:description] unless (content[:description].nil? || content[:description].empty?)
+        use = (config_file == @previous_scenario ? 'INSTALLED' : "use: --scenario #{scenario}")
```

Comment:
detection does not work when relative path is used, I guess `File.expand_path` should be used